packages/textfield/src/Textfield.ts

```diff
@@ -303,11 +334,10 @@ export class TextfieldBase extends ManageHelpText(
                 : nothing}
             <!-- @ts-ignore -->
             <textarea
+                id="field"
                 name=${ifDefined(this.name || undefined)}
                 aria-describedby=${this.helpTextId}
-                aria-label=${this.label ||
-                this.appliedLabel ||
```

Comment:
I noticed there was a `TODO` for deprecating `appliedLabel` in (maybe?) combobox. Did we want or need to do that here as well?